web/src/info_overlay.ts

```diff
@@ -36,115 +36,106 @@ function format_usage_html(...keys: string[]): string {
     },
     {
         markdown: ":heart:",
     },
     {
-        markdown: "[Zulip website](https://zulip.org)",
+        markdown: `[${$t({defaultMessage: "Zulip website"})}](https://zulip.org)`,
         usage_html: format_usage_html("Ctrl", "Shift", "L"),
     },
     {
-        markdown: "#**channel name**",
-        output_html: "<p><a>#channel name</a></p>",
-        effect_html: "(links to a channel)",
+        markdown: `#**${$t({defaultMessage: "channel name"})}**`,
+        output_html: `<p><a>#${$t({defaultMessage: "channel name"})}</a></p>`,
+        effect_html: $t({defaultMessage: "(links to a channel)"}),
     },
     {
-        markdown: "#**channel name>topic name**",
-        output_html: "<p><a>#channel name > topic name</a></p>",
-        effect_html: "(links to topic)",
+        markdown: `#**${$t({defaultMessage: "channel name"})}>${$t({defaultMessage: "topic name"})}**`,
+        output_html: `<p><a>#${$t({defaultMessage: "channel name"})} > ${$t({defaultMessage: "topic name"})}</a></p>`,
+        effect_html: $t({defaultMessage: "(links to topic)"}),
     },
     {
-        markdown: "@**Joe Smith**",
-        output_html:
-            '<p><span class="user-mention"><span class="mention-content-wrapper">@Joe Smith</span></span></p>',
-        effect_html: "(notifies Joe Smith)",
+        markdown: `@**${$t({defaultMessage: "Joe Smith"})}**`,
+        effect_html: $t({defaultMessage: "(notifies Joe Smith)"}),
     },
     {
-        markdown: "@_**Joe Smith**",
-        output_html:
-            '<p><span class="user-mention"><span class="mention-content-wrapper">Joe Smith</span></span></p>',
-        effect_html: "(links to profile but doesn't notify Joe Smith)",
+        markdown: `@_**${$t({defaultMessage: "Joe Smith"})}**`,
+        effect_html: $t({defaultMessage: "(links to profile but doesn't notify Joe Smith)"}),
     },
     {
-        markdown: "@*support team*",
-        // Since there are no mentionable user groups that we can
-        // assume exist, we use a fake data-user-group-id of 0 in
-        // order to avoid upsetting the rendered_markdown.ts
-        // validation logic for user group elements.
-        //
-        // Similar hackery is not required for user mentions as very
-        // old mentions do not have data-user-id, so compatibility
-        // code for that case works ... but it might be better to just
-        // user your own name/user ID anyway.
-        output_html:
-            '<p><span class="user-group-mention" data-user-group-id="0"><span class="mention-content-wrapper">@support team</span></span></p>',
-        effect_html: "(notifies <b>support team</b> group)",
+        markdown: `@*${$t({defaultMessage: "support team"})}*`,
+        effect_html: $t_html(
+            {defaultMessage: "(notifies <user-group></user-group> group)"},
+            {"user-group": () => `<b>${$t({defaultMessage: "support team"})}</b>`},
```

Comment:
I think there's a better way to do this, where `support team` is inside the `<z-user-group>` tags. Something like this:
``` diff
     {
         markdown: `@*${$t({defaultMessage: "support team"})}*`,
         effect_html: $t_html(
-            {defaultMessage: "(notifies <user-group></user-group> group)"},
-            {"user-group": () => `<b>${$t({defaultMessage: "support team"})}</b>`},
+            {defaultMessage: "(notifies <z-user-group>support team</z-user-group> group)"},
+            {"z-user-group": (content_html) => `<b>${content_html.join("")}}</b>`},
         ),
     },
     {
```

but I guess the risk of that is that the group names might not match. OK, maybe this is broadly the right way to do it.